src/Mirza/Common/Utils.hs

```diff
@@ -137,3 +155,36 @@ addLastUpdateTriggers db = forM_ (getTableNames db) $ \tName -> do
       "CREATE TRIGGER sync_lastmod \
       \BEFORE UPDATE OR INSERT ON \"" <> tName <>
         "\" FOR EACH ROW EXECUTE PROCEDURE sync_lastmod();"
+
+{-# NOINLINE manager' #-}
+manager' :: C.Manager
+manager' = unsafePerformIO $ C.newManager C.defaultManagerSettings
```

Comment:
I realise you didn't write this, but why not include the http manager in the `context` for the AppM monad and call `C.newManager` when we do the rest of the initialisation.

Reply:
We could, but then that would mean that we need to include separate `manager`s for `SCSContext` and `BRContext`, and then `runClient` the function won't remain generic anymore.
Correct me if I am wrong, but `unsafePerformIO` should be fine in this case since we are using `defaultManagerSettings`.

Reply:
It is fine and not unusual to see in Haskell code wanting to create a global variable set to the result of an IO action.

It was more of a nitpick in that there are other ways to do it, and since we are already passing around the `context` it could have been done that way to avoid unsafePerformIO. Regarding `runClient`, it'd be possible to keep it generic using mtl-style constraints, e.g.

`runClient :: (MonadIO m, MonadReader c m, HasHttpManager c Manager) -> BaseUrl -> ClientM a -> m (Either ServantError a)` 

In any case, not really a problem, leave as is for now.